### PR TITLE
build via CircleCI and add status badge to readme

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-        - cd $GS_WD && go test -v $(glide novendor)
-        - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
+  override:
+    - ./architect build
+
+deployment:
+  master:
+    branch: master
+    commands:
+      - ./architect deploy
```
Comment (xh3b4sd) [inline circle.yml]: @JosephSalisbury how do we want to roll with this? The lib will not be deployed anyway. Should it anyway stay? WDYT?

File: circle.yml [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 dependencies:
-    override:
-        - sudo add-apt-repository ppa:masterminds/glide -y
-        - sudo apt-get update
-        - sudo apt-get install glide -y
-        - mkdir -p $HOME/go/src/github.com/$CIRCLE_PROJECT_USERNAME
-        - cp -rf $HOME/$CIRCLE_PROJECT_REPONAME $HOME/go/src/github.com/$CIRCLE_PROJECT_USERNAME
+  override:
+    - |
+      wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
```
Comment (xh3b4sd) [inline circle.yml]: Where do I get the release token from?

File: circle.yml (hunk 1 above)
Comment (JosephSalisbury) [inline circle.yml]: I'd rather keep it, for my inner monk. We may have some deploy step for libraries in the future.

File: circle.yml (hunk 2 above)
Comment (JosephSalisbury) [inline circle.yml]: keepass - taylorbot, see comment

**Comment by xh3b4sd** [comment]
Fishing for LGTM.

**Pull request merged**